src/include/machnet_pkthdr.h

```diff
@@ -19,10 +19,11 @@ struct __attribute__((packed)) MachnetPktHdr {
   be16_t magic;  // Magic value tagged after initialization for the flow.
   enum class MachnetFlags : uint8_t {
     kData = 0b0,
-    kSyn = 0b1,         // SYN packet.
-    kAck = 0b10,        // ACK packet.
-    kSynAck = 0b11,     // SYN-ACK packet.
-    kRst = 0b10000000,  // RST packet.
+    kSyn = 0b1,            // SYN packet.
+    kAck = 0b10,           // ACK packet.
+    kSynAck = 0b11,        // SYN-ACK packet.
+    kRst = 0b10000000,     // RST packet.
+    kRetryForRss = 0b100,  // Retry packet.
```

Comment:
This should go before `kRst`. Comment should be more descriptive `Retry for RSS packet.`.